plugins/lua/ts_lua.c

```diff
@@ -26,36 +26,311 @@
 
 #define TS_LUA_MAX_STATE_COUNT 256
 
+#define TS_LUA_STATS_TIMEOUT 1000   // 1s
```

Comment:
can this be configurable as well? 
I think this may be too much initially. Perhaps a higher number as the default? like 5s ?
Also it may be a good way to disable stats if the timeout is set to 0? 

That said, I think we can still merge this first and submit another PR later for this.